resources/definitions/agile.def.json

```diff
@@ -14,7 +14,11 @@
     "variants_name": "Nozzle",
     "preferred_variant_name": "0.4 mm",
     "preferred_material": "pro_pla",
-    "supports_usb_connection": false
+    "supports_usb_connection": false,
+    "machine_extruder_trains":
+       {
+            "0": "agile_extruder_0"
```

Comment:
Mmmmm, but this is not enough. This "agile_extruder_0" makes a reference to an existing file within the `extruders` folder. Currently you didn't provide it, so you need to create this file with the right data in it.

Reply:
resources/extruders/agile_extruder_0.def.json

Reply:
https://github.com/Ultimaker/Cura/compare/master...robintiger:patch-8#diff-239b5bd0f47f2103162a15bb152f0303